### DISCUSSION TITLE
Animated unicode plot inside Term

### DISCUSSION THREAD

**Discussion by zsoerenm** [FedeClaudi/Term.jl#145] [Q&A] [answered]
I'd like to have an animated Unicode plots inside Term.
Would something like the following be possible in Term?
https://discourse.julialang.org/t/ann-unicodeplots-jl-v2-8-0-3d-plots/75963/13?u=zsoerenm

**Comment by FedeClaudi** [comment, marked as answer]
Hi, 

The way the example animation you've sent is done is to repeatedly: 1) clear out the terminal and 2) print an updated display. 
You can do the same with anything you'd print to the REPL, including things made with term:
```julia
using Term
import Term.Consoles: clear

make_p(i) = Panel("Iteration $i", fit=true)

for i in 1:10
    clear()
    make_p(i) |> print
    sleep(1)
end
```

This might look a bit glitchy if the function in the loop takes a while or if actually printing the content takes a while (e.g. if Term needs to do a lot of work). One solution might be to use `sprint` or `string` to first generate some `content` and then call `clear(); print(content)` to avoid delays between clearing and printing. 

Hope this help. 


Ps: I will move this to a discussion to make it more discoverable for other people interested in similar things.
